### ISSUE TITLE
Banner "Icon Packs"

### ISSUE THREAD

**Issue by Fleker** [ITVlab/TvAppRepo#39]
What if I could generate shortcut apps with banners that come from third-party apps, similar to icon packs?

**Comment by Xstar97** [comment]
so third party devs can generate & share banner/logos png's to your app for your users to use?

**Comment by Fleker** [comment]
Yeah it'd be interesting if others can build icon packs that include banners
